File: cylinder.py
# Created by : Sidney Kang
# Created on : 6 Nov. 2017
# Created for : ICS3UR
# Daily Assignment - Unit4-05
# This program calculates area of cylinder

import math

def calculate_cylinder_area(height, radius):
	
    #constant
    constant = 2 * math.pi
    volume_of_cylinder = float(height * constant * (radius ** 2))
    
    return volume_of_cylinder
    
height = raw_input("Enter the height of the cylinder: ")
height = int(height)
radius = raw_input("Enter the radius of the cylinder: ")
radius = int(radius)

calculate_cylinder_area(height, radius)

volume = calculate_cylinder_area(height, radius)
print("Volume is: " + str(volume) + " units^3")

File: Cylinder.java
/****************************************************************************
 *
 * Created on : 20 Dec. 2017
 * Created for : ICS3UR
 * Daily Assignment - Unit 4-05
 * This program calculates area of cylinder
 *
 ****************************************************************************/
import java.util.Scanner;
import java.lang.Math;

public class Cylinder
{
     static double CONSTANT = 2 * Math.PI;
     
     public static double cylinderVolume(double height, double radius)
     {          
           double volume = height * CONSTANT * (java.lang.Math.pow(radius, 2));
    
           return volume;
       }
  
     public static void main(String[] args)
     { 
          try 
          {               
               double height, radius;
               Scanner input = new Scanner(System.in);  
               System.out.println("Enter height of cylinder: ");
               height = input.nextDouble();
               System.out.println("Enter radius of cylinder: ");
               radius = input.nextDouble();      
      
               double volume = cylinderVolume(height, radius);
               System.out.println("The volume of cylinder is: " + volume + " units^3");              
           } 
           catch(Exception nfe) 
           {
                System.err.println("Invalid input!");
           }            
      }
}
